src/main/java/mate/academy/spring/service/impl/AuthenticationServiceImpl.java

```diff
@@ -10,18 +13,23 @@
 public class AuthenticationServiceImpl implements AuthenticationService {
     private final UserService userService;
     private final ShoppingCartService shoppingCartService;
+    private final RoleService roleService;
 
     public AuthenticationServiceImpl(UserService userService,
-                                     ShoppingCartService shoppingCartService) {
+                                     ShoppingCartService shoppingCartService,
+                                     RoleService roleService) {
         this.userService = userService;
         this.shoppingCartService = shoppingCartService;
+        this.roleService = roleService;
     }
 
     @Override
     public User register(String email, String password) {
         User user = new User();
+        Role userRole = roleService.getByName(String.valueOf(Role.RoleName.USER));
```

Comment:
You can use RoleName.USER.name() here
```suggestion
        Role userRole = roleService.getByName(Role.RoleName.USER.name());
```